s3tests_boto3/functional/test_s3.py

```diff
@@ -7870,6 +8458,34 @@ def test_lifecycle_expiration():
     eq(len(keep2_objects), 4)
     eq(len(expire3_objects), 2)
 
+def test_lifecyclev2_expiration():
```

Comment:
This test really needs a `@attr('lifecycle_expiration')` like the rest of the lifecycle expiration tests.

Reply:
Added.